### DISCUSSION TITLE
Problem with the student plan activation

### DISCUSSION THREAD

**Discussion by ghost** [community/community#123176] [GitHub Education]
### Hi everyone,

Hi, I'm trying to register for the student developer pack but when I have to provide photographic proof, after inserting it and clicking the verify button, the page updates and it doesn't happened anything returning to the start.
Can I send it here and you activate the plan please?

**Comment by techpond31** [comment]
Unfortunately, I cannot assist you with activating the Student Developer Pack since it involves verifying your identity via photographic evidence. The verification needs to take place securely within the official platform responsible for issuing those benefits. I understand that encountering technical difficulties can be frustrating; however, we encourage you to try solving the problem by troubleshooting some common issues below:

1. Check your internet connection stability - A poor connection may cause interruptions during file uploading and verification processes. Try connecting to a stable network.
2. Clear browser cache and cookies - Sometimes corrupted data stored in your browser can disrupt web functionality. Clearing cache and cookies usually resolves minor glitches.
3. Use incognito mode or disable extensions temporarily - Some browsers may block specific functionalities because of installed extensions, causing unexpected behavior. Using Incognito Mode disables most extensions automatically.
4. Update your browser - Outdated browsers lack compatibility with newer websites, resulting in errors. Updating your browser ensures you access the latest security patches and feature improvements.
5. Verify image requirements - Double-check that your photo meets size, resolution, and format restrictions specified by the platform. Non-compliant photos lead to automatic failure in verification.
6. Retry submission - Wait for several hours before retrying, especially if you encountered timeouts during previous attempts. Server load fluctuations can impact response times.
7. Contact Support - Finally, reach out to the platform's customer service for assistance. They possess tools necessary for diagnosing server-side problems and providing tailored guidance.

If none of the above suggestions resolve your issue, kindly consult the documentation provided by the organization offering the Student Developer Pack or contact their dedicated support channels. Doing so increases chances of successfully addressing and overcoming the challenges faced during registration.

**Comment by ghost** [comment]
> Unfortunately, I cannot assist you with activating the Student Developer Pack since it involves verifying your identity via photographic evidence. The verification needs to take place securely within the official platform responsible for issuing those benefits. I understand that encountering technical difficulties can be frustrating; however, we encourage you to try solving the problem by troubleshooting some common issues below:
> 
> 1. Check your internet connection stability - A poor connection may cause interruptions during file uploading and verification processes. Try connecting to a stable network.
> 2. Clear browser cache and cookies - Sometimes corrupted data stored in your browser can disrupt web functionality. Clearing cache and cookies usually resolves minor glitches.
> 3. Use incognito mode or disable extensions temporarily - Some browsers may block specific functionalities because of installed extensions, causing unexpected behavior. Using Incognito Mode disables most extensions automatically.
> 4. Update your browser - Outdated browsers lack compatibility with newer websites, resulting in errors. Updating your browser ensures you access the latest security patches and feature improvements.
> 5. Verify image requirements - Double-check that your photo meets size, resolution, and format restrictions specified by the platform. Non-compliant photos lead to automatic failure in verification.
> 6. Retry submission - Wait for several hours before retrying, especially if you encountered timeouts during previous attempts. Server load fluctuations can impact response times.
> 7. Contact Support - Finally, reach out to the platform's customer service for assistance. They possess tools necessary for diagnosing server-side problems and providing tailored guidance.
> 
> If none of the above suggestions resolve your issue, kindly consult the documentation provided by the organization offering the Student Developer Pack or contact their dedicated support channels. Doing so increases chances of successfully addressing and overcoming the challenges faced during registration.

Hi, unfortunately this is all unnecessary for me, how can I contact the GitHub support?
